cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -218,3 +235,7 @@ private def getSkippedPaths(String[] repos) {
     log.info "Elapsed time to retrieve paths to skip: " + duration
     return skip
 }
+
+private def checkName(keepRelease, releaseRegex, artifactName) {
```

Comment:
I think the function name is as well not good enough. The function returns a boolean so the name should somehow indicate that (similar to canDelete, isAvailable, etc.)